src/Core/SettingsChangesHistory.cpp

```diff
@@ -768,7 +768,7 @@ const VersionToSettingsChangesMap & getMergeTreeSettingsChangesHistory()
     {
         addSettingsChanges(merge_tree_settings_changes_history, "25.7",
         {
-
+            {"input_format_protobuf_oneof_presence", false, false, "New setting"},
```

Comment:
Tests are failing because this setting was added to MergeTree settings history, please, move it to the usual settings hostory above